lib/controllers/zodiacs.js

```diff
@@ -0,0 +1,14 @@
+const { Router } = require('express');
+const { zodiacs } = require('../zodiacs-data');
+
+module.exports = Router()
+  .get('/:id', (req, res) => {
+    const match = zodiacs.find(({ id }) => id === req.params.id);
+    return res.json(match);
+  })
+  .get('/', (req, res) => {
+    const filteredData = zodiacs.map((zodiac) => {
+      return { id: zodiac.id, name: zodiac.name };
+    });
+    res.json(filteredData);
+  });
```

Comment:
ideally there's a commit for test, for route, for next test, for next route, etc.